lib/resources/host.rb

```diff
@@ -48,6 +48,8 @@ def initialize(hostname, params = {})
         @host_provider = LinuxHostProvider.new(inspec)
       elsif inspec.os.windows?
         @host_provider = WindowsHostProvider.new(inspec)
+      elsif inspec.os.darwin?
```

Comment:
I just copied this behavior from the windows resource, but yeah I see what you're saying.. Going to put a `skip_resource` in the beginning of the initialize for UDP since this isn't supported on any platform if that makes sense.